### DISCUSSION TITLE
Help deploying Dapp

### DISCUSSION THREAD

**Discussion by superserrano** [hashlips-lab/nft-erc721-collection#223] [Q&A] [answered]
When I deploy my Dapp on Fleek or Netlify, the Dapp does not show up on the website, but everything else looks great.  Everything works flawlessly on my local server.
I have tried the solution mentioned on your Discord: add homepage:"." to the package.json and tried manually setting the homepage with my url.  No go.
My deployment settings on fleek are:
  Base directory: minting-dapp
  Build command: yarn
  Publish directory: /public

Anyone experiencing similar issues and have worked through this?  Any help would be greatly appreciated.

**Comment by superserrano** [comment]
Got it!  All was working fine: the Dapp, page etc.  What happened was that the entire contents of the minting-dapp/public/build folder did not get committed  and pushed to my github repository.

**Comment by liarco** [comment, marked as answer]
Hi @superserrano,
committing the build files to your repo is not exactly a best practice, you should avoid this (files in that directory are ignored by git on purpose). I suggest you taking a look at #221. ;)

A video tutorial about this should be published soon on my YouTube channel.

**Reply by superserrano** [reply]
Thank for the heads up, liarco.  Will be addressing that.  Btw, thanks for the all of the work you have done here.  We owe you a debt of gratitude.
